### ISSUE TITLE
Quantiazation with GeoJSON output

### ISSUE THREAD

**Issue by hyperknot** [topojson/topojson#334]
I've been following the Command-Line Cartography tutorial and one thing doesn't make sense for me.

It runs `topoquantize` and then it outputs the file as a GeoJSON at the end. I don't understand this. Isn't topojson quantization a space saving technique only in the TopoJSON format? 

I believe as soon as we convert to GeoJSON, it'll actually increate space usage, because of the rational number's floating point conversion (or I don't understand how it works).

I mean in GeoJSON everything is floating point in text format, so the only kind of quantization which would make sense there would be cutting down decimal digits. Currently topo2geo creates coordinates like 22.779227792277936, with 15 decimal digits. This [super famous answer](https://gis.stackexchange.com/a/8674/52450) only lists until 9 digits, which is on the scale of microns.

Short version, what is the best version to compress GeoJSON > GeoJSON while handling topology / borders properly.

**Comment by mbostock** [comment]
Quantization reduces the size of the TopoJSON, but can also improve the quality of the topology if the input coordinates are messy. This is why there is both pre-quantization (before the topology is computed) and post-quantization (after).

> It runs topoquantize and then it outputs the file as a GeoJSON at the end.

The tutorial is focused on the command-line and producing a static SVG, but often you want to fetch the TopoJSON file in the browser to render it dynamically, in which case using TopoJSON as the transport format is more efficient than GeoJSON.

The tutorial uses [geo2svg -p](https://github.com/d3/d3-geo-projection#geo2svg_precision) to limit the precision of the coordinates in the output SVG (in pixels).

If you want to round the coordinates of the GeoJSON output produced by topo2geo, you can do that using [ndjson-cli](https://github.com/mbostock/ndjson-cli) or similar. I didn’t implement a -p flag for topo2geo because my recommendation is to use TopoJSON rather than GeoJSON as a transport format.

**Comment by hyperknot** [comment]
Thanks for the answer. What settings do you recommend for the "if the input coordinates are messy" case? 10e8 or 10e10?

**Comment by hyperknot** [comment]
For example Natural Earth 10m level dataset.
